base/docs/helpdb/Base.jl

```diff
@@ -207,7 +207,10 @@ isnan
 """
     println(x)
 
-Print (using [`print`](:func:`print`)) `x` followed by a newline.
+Print (using [`print`](:func:`print`)) `x` followed by a newline. To print to a specific output
```

Comment:
I was using the docs for `show` as a reference, which doesn't do that either. Maybe it's intended? And I agree that showing it in the description is clearer.